docs/rst/getting_started/intro.rst

```diff
@@ -37,4 +38,11 @@ Perform simulations on three-dimensional scenes.
     Eradiate supports simulations on 3D scenes consisting of a vegetated ground
     patch with no atmosphere above it.
 
+Running with Docker
+-------------------
+
+Docker users may benefit from the images provided by the Eradiate team.
+
+Those images allow a fast setup and can provide a ready to use Jupyter Lab server for experimenting.
+
```

Comment:
Should I make this part of the [Installation guide](https://eradiate.readthedocs.io/en/latest/rst/getting_started/install.html) page instead? This page seems to be more suitable to answer the question "How to install Eradiate".

I could rewrite the docker setup as an alternative installation procedure and include it in this page. Would this be more suitable?

Reply:
I think it would easier to find the Docker content if it has its own page: it could contain both the setup and usage tips (a lot of our users are not experienced with Docker and will probably need guidance). Then, a note in the installation guide would redirect to this page.

If this proves to be impractical in the future, we'll reshuffle the content.

Reply:
Ok good. I will rework this piece of doc the following way:
 - Include more examples/use cases and tips
 - Add more details on the docker installation procedure
 - Rename it Docker guide
 - Add a section redirecting to it in the installation guide

This will follow some rework of the Jupyterlab image to include a procedure for working with multiple jupyter instances.

Reply:
Hello Vincent, the changes listed above have been made, and are ready for revew :)